modules/cli/src/main/scala/scala/cli/commands/InstallCompletions.scala

```diff
@@ -114,4 +84,33 @@ object InstallCompletions extends ScalaCommand[InstallCompletionsOptions] {
           System.err.println(s"$rcFile already up-to-date")
     }
   }
+
+  def getName(name: Option[String]) =
+    name.getOrElse {
+      val baseName = (new Argv0).get("scala-cli")
+      val idx      = baseName.lastIndexOf(File.separator)
+      if (idx < 0) baseName
+      else baseName.drop(idx + 1)
+    }
+
+  def getFormat(format: Option[String], interactive: Interactive, command: String) =
```

Comment:
I think a function like this should not run `sys.exit(1)`. For example, getFormat can return `Option[String]` and it could be used safely in other places.